stream.go

```diff
@@ -171,6 +190,55 @@ type StreamSignedURLParameters struct {
 	AccessRules  []StreamAccessRule `json:"accessRules,omitempty"`
 }
 
+type StreamInitiateTUSUploadParameters struct {
+	DirectUserUpload bool
+	TusResumable     TusProtocolVersion
+	UploadLength     int64
+	UploadCreator    string
+	Metadata         TUSUploadMetadata
+}
```

Comment:
```suggestion
type StreamInitiateTUSUploadParameters struct {
	DirectUserUpload bool               `url:"direct_user,omitempty"`
	TusResumable     TusProtocolVersion `url:"-"`
	UploadLength     int64              `url:"-"`
	UploadCreator    string             `url:"-"`
	Metadata         TUSUploadMetadata  `url:"-"`
}
```

Reply:
this allows us to use the struct for query parameter building in `buildURI` instead of manually performing it.